Tools/MiniBrowserSwiftUI/WebKit_SwiftUI/WebView.swift

```diff
@@ -104,6 +104,10 @@ private struct _WebView {
         view.uiDelegate = coordinator
         coordinator.webView = view
         coordinator.environment = environment
+        
+        // Configuration Options
```

Comment:
IMO, don't really need the comment. How does this work though? Does the `WKWebView` just internally vend some keyboard shortcuts or something that allows this to work, or how would the user actually effect this?

Is the intent here that for API we'd have these enabled by default from `WebView` even though they're opt-in for `WKWebView`?